packages/@aws-cdk/aws-codebuild/test/test.project.ts

```diff
@@ -1,4 +1,13 @@
-import { countResources, expect, haveResource, haveResourceLike, objectLike, not, ResourcePart, arrayWith } from '@aws-cdk/assert';
+import {
+  arrayWith,
+  countResources,
+  expect,
+  haveResource,
+  haveResourceLike,
+  not,
+  objectLike,
+  ResourcePart,
+} from '@aws-cdk/assert';
```

Comment:
Any chance we can keep this as-is?

Reply:
Reverted :+1: , I guess it formated with my IDE :-)